tests/stubtest_allowlists/py3_common.txt

```diff
@@ -392,6 +393,13 @@ email.contentmanager.set_bytes_content
 email.contentmanager.set_message_content
 email.contentmanager.set_text_content
 email.iterators.walk
+pipes.Template.makepipeline
+pipes.Template.open_r
+pipes.Template.open_w
+sunau.Au_read.initfp
+sunau.Au_write.initfp
+wave.Wave_read.initfp
+wave.Wave_write.initfp
```

Comment:
These seem to be implementation details that just happen to not be named with `_` in front. It might be good to add a comment that mentions this.

Reply:
This whole section of the allowlist is subtitled "undocumented implementation details"